lib/mnemosyne/middleware/rack.rb

```diff
@@ -109,7 +109,10 @@ def scan_response(trace, response)
 
         return unless headers
 
-        trace.meta[:content_type] = headers['Content-Type']
+        trace.meta.merge!({
+          content_type: headers['Content-Type'],
```

Comment:
When `Content-Type` header is missing, this item will no longer be attached. I suppose this should be fine?